index.js

```diff
@@ -13,60 +25,117 @@ const QUESTIONS = [{
     }]
 }];
 
+// Use the body-parser middleware to parse the request body
+app.use(bodyParser.json())
 
-const SUBMISSION = [
-
-]
+const SUBMISSIONS = [];
 
 app.post('/signup', function(req, res) {
+  const USERS = []
   // Add logic to decode body
-  // body should have email and password
-
+  const { email, password, isAdmin } = req.body
 
-  //Store email and password (as is for now) in the USERS array above (only if the user with the given email doesnt exist)
+  // Check if the user with the given email already exists
+  const userExists = USERS.some(user => user.email === email)
+  if (userExists) {
+    // If the user already exists, return 409 Conflict status code
+    res.status(409).send('User already exists!')
+    return
+  }
 
+  // Store email and password in the USERS array
+  USERS.push({ email, password, isAdmin })
 
-  // return back 200 status code to the client
-  res.send('Hello World!')
-})
+  // Return back 200 status code to the client
```

Comment:
should be 201, because an user object is created in the db

Reply:
there is no DB yet!

Reply:
By db I mean wherever you're storing the data